panel/src/components/Lab/PlaygroundView.vue

```diff
@@ -113,13 +113,6 @@ export default {
 </script>
 
 <style>
-.k-lab-examples h2 {
-	margin-bottom: var(--spacing-6);
-}
-.k-lab-examples * + h2 {
-	margin-top: var(--spacing-12);
-}
```

Comment:
`Todo:` This removal breaks parts of the Lab, e.g. https://sandbox.test/panel/lab/internals/panel/api and other of its tabs.

Reply:
Ah, good that you spotted the issue. I was not able to find where this rule applied. I will fix it.